src/components/ServerSdkContext.tsx

```diff
@@ -91,12 +85,17 @@ export const ServerSDKProvider = ({ children }: Props) => {
     () => (httpApiUrl ? new PeerApi(undefined, httpApiUrl || "", client) : null),
     [client, httpApiUrl],
   );
+  const componentApi = useMemo(
+    () => (httpApiUrl ? new ComponentApi(undefined, httpApiUrl || "", client) : null),
```

Comment:
```suggestion
    () => (httpApiUrl ? new ComponentApi(undefined, httpApiUrl, client) : null),
```
httpApiUrl will not be falsy in this place